packages/next-swc/crates/next-api/src/project.rs

```diff
@@ -254,6 +282,20 @@ impl Project {
         self.project_fs().root()
     }
 
+    /// Returns a path to dist_dir resolved from project root path as string.
+    /// Currently this is used to embed process.env.__NEXT_DIST_DIR.
+    /// [Note] in webpack-config, it is being injected when
+    /// dev && (isClient || isEdgeServer)
+    #[turbo_tasks::function]
+    async fn dist_root_string(self: Vc<Self>) -> Result<Vc<String>> {
+        let this = self.await?;
+        Ok(self
+            .project_root_path()
+            .root()
+            .join(this.dist_dir.to_string())
+            .to_string())
+    }
```

Comment:
This is just `node_root().to_string()`, no?

Reply:
prolly yes, seems I missed that.